0002-gitops-management-clusters/0002-gitops-management-clusters.md

```diff
@@ -0,0 +1,27 @@
+# RFC 0002 - Enable customers to use gitops in management clusters
+
+We currently give customers access to the management clusters but do not support them in utilizing this access effectively in terms of git ops related management (e.g. for apps).
+
+Context:
+Some customers might be interested in utilizing mainstream git ops tooling in order to manage their apps across clusters and installations.
+
+## Open Questions
+
+### 1. General direction
+1.1 We would like to allow our customers to choose their management tooling freely. Do we want to support some tools with e.g. a managed app first though?
+
+1.2 Do we own the gitops tooling or is it purely owned by the customer?
+
+### 2. Technical issues
+2.1 How does the setup for an in-cluster agent look like? Currently customers will struggle setting up an in-cluster agent with appropriate permissions.
+
+2.2 How do we ensure security requirements when customer interaction increases?
```

Comment:
There is a multi tenancy problem with chart CRs and their related configmaps and secrets. 

These are currently stored in the giantswarm namespace. Before enabling customers to create apps in management clusters I think we should move this to their organization or cluster namespace.